mmdet/models/utils/resize.py

```diff
@@ -0,0 +1,21 @@
+from torch.nn import functional as F
+
+
+def up_sample_like(logits, target):
+    assert len(target.shape) >= 2
```

Comment:
1. might need to expose mode and align_corners, or directly pass *args and **kwards to F.interpolate
2. docstring
3. upsample_like